posts/2019/11/metadsl-talk.md

```diff
@@ -0,0 +1,207 @@
+<!--
+.. title: metadsl PyData Talk
+.. slug: metadsl-talk
+.. date: 2019-12-10 12:00:00 UTC-05:00
+.. author: Saul Shanabrook
+.. tags: Labs, metadsl
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+ 
+# [`metadsl`](https://metadsl.readthedocs.io/en/latest/) talk
```

Comment:
This doesn't work, for some reason that ate up all the content under the `#` and `##` sections. Pushed a fix.

Reply:
Thanks!